src/components/Gauge/Gauge.scss

```diff
@@ -0,0 +1,76 @@
+@import '../../themes/oneui/oneui.scss';
+
+.Gauge {
+    position: relative;
+
+    &__svg {
+        transform: rotate(180deg);
+    }
+
+    &__circleBackground {
+        stroke: var(--color-neutral-25);
+        stroke-width: var(--spacing-2x);
+        stroke-linecap: round;
+        fill: none;
+    }
+
+    &__circleForeground {
+        stroke-width: var(--spacing-4x);
+        stroke-linecap: round;
+        fill: none;
+        transition: all var(--transition-duration) ease-out;
+
+        &--isProgressLoading {
+            opacity: 0;
+        }
+    }
+
+    &__contentPlaceholder {
+        margin-bottom: 0;
+    }
+
+    &__contentWrapper {
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        margin-top: var(--spacing-normal);
+        transform: translate(-50%, -100%);
+        text-align: center;
+
+        &--isContentLoading {
+            width: 100px;
+        }
+    }
+
+    &__content {
+        font-size: var(--h1-font-size);
+        font-weight: var(--font-weight-bold);
+    }
+
+    &__metric {
+        display: block;
+        font-size: var(--font-size-normal);
+        font-weight: var(--font-weight-normal);
```

Comment:
Do we need additional styling in case the text here is too long to fit into one line (e.g. some German translation)? @carlobernardini

Reply:
I suggest you force it to be on one line (`white-space: nowrap`) and truncate when too long (`overflow: hidden` and `text-overflow: ellipsis`). You can add e.g. a `title` attribute so you still see the full thing on hover.

Reply:
@carlobernardini Does it mean that `metric` prop should always be a string?

Reply:
Yes, that can always be a string